deployment/deployment.py

```diff
@@ -13,15 +13,17 @@ def print_command(command):
 def execute_ssh_command(ssh_client, command):
     print_command(command)
     stdin, stdout, stderr = ssh_client.exec_command(command)
-    for line in stdout.readlines():
-        sys.stdout.write(line)
-    for line in stderr.readlines():
-        sys.stdout.write(line)
+    # for line in stdout.readlines():
+    #    sys.stdout.write(line)
+    # for line in stderr.readlines():
+    #    sys.stdout.write(line)
     return
 
 CONFIG_FILE = 'deployment.cnf'
+SERVER_LIST_FILE = 'servers.txt'
+METADATA_FILE = 'metadata.db'
 PREFIX = '../'
-FILES = [ 'server.py', 'client.py', 'util.py', 'requirements.txt' ]
+FILES = [ 'server.py', 'client.py', 'metadata_manager.py', 'util.py', 'requirements.txt', 'metadata.sql', SERVER_LIST_FILE ]
```

Comment:
could we perhaps rename this `FILES_TO_DEPLOY` or something a little more descriptive?